spec/support/seed_repo.rb

```diff
@@ -9,7 +9,7 @@
 # fa1b1e6c004a68b7d8763b86455da9e6b23e36d6 Merge branch 'add-files' into 'master'
 # eb49186cfa5c4338011f5f590fac11bd66c5c631 Add submodules nested deeper than the root
 # 18d9c205d0d22fdf62bc2f899443b83aafbf941f Add executables and links files
-# 5937ac0a7beb003549fc5fd26fc247adbce4a52e Add submodule from gitlab.com
```

Comment:
This change seems odd.

Reply:
Oh, we didn't register `bringit.com` yet? :smile: